src/components/SearchToolbar/SearchResults.js

```diff
@@ -21,6 +21,7 @@ import type { SearchResultFeature, SearchResultCollection } from '../../lib/sear
 
 type SearchResultProps = {
   result: SearchResultFeature,
+  history: RouterHistory,
```

Comment:
Maybe: Router dependency in component, might move this up to app level. This will need to be refactored once we merge the apps.